test/index.js

```diff
@@ -1372,6 +1374,76 @@ describe('Nightmare', function () {
       yield nightmare.end();
     });
 
+    /* screenshot jpeg */
+    it('should take a jpeg screenshot', function*() {
+      yield nightmare
+        .goto('https://github.com/')
+        .screenshot(tmp_dir+'/test.jpg');
+      var stats = fs.statSync(tmp_dir+'/test.jpg');
+      var fileChunk = readChunk.sync(tmp_dir+'/test.jpg',0,4100);
+      var mime = fileType(fileChunk).mime;
+      mime.should.equal('image/jpeg');
+      stats.size.should.be.at.least(1000);
+    });
+
+    it('should buffer a jpeg screenshot', function*() {
+      var image = yield nightmare
+        .goto('https://github.com')
+        .screenshot(90);
+      var stats = fs.statSync(tmp_dir+'/test.jpg');
+      Buffer.isBuffer(image).should.be.true;
+      var mime = fileType(image).mime;
+      mime.should.equal("image/jpeg");
+      image.length.should.be.at.least(1000);
+    });
+    it('should take a clipped jpeg screenshot', function*() {
+      yield nightmare
+        .goto('https://github.com/')
+        .screenshot(tmp_dir+'/test-clipped.jpg',5, {
+          x: 200,
+          y: 100,
+          width: 100,
+          height: 100
+        });
+      var stats = fs.statSync(tmp_dir+'/test.jpg');
+      var statsClipped = fs.statSync(tmp_dir+'/test-clipped.jpg');
+      statsClipped.size.should.be.at.least(300);
+      stats.size.should.be.at.least(10*statsClipped.size);
+    });
+    it('should take a clipped jpeg screenshot with default compression', function*() {
+      yield nightmare
+        .goto('https://github.com/')
+        .screenshot(tmp_dir+'/test-clipped.jpg', {
+          x: 200,
+          y: 100,
+          width: 100,
+          height: 100
+        });
+      var stats = fs.statSync(tmp_dir+'/test.jpg');
+      var statsClipped = fs.statSync(tmp_dir+'/test-clipped.jpg');
+      statsClipped.size.should.be.at.least(300);
+      stats.size.should.be.at.least(10*statsClipped.size);
+    });
+
+    it('should buffer a clipped jpeg screenshot', function*() {
+      var image = yield nightmare
+        .goto('https://github.com')
+        .screenshot(90,{
+          x: 200,
+          y: 100,
+          width: 100,
+          height: 100
+        });
+      Buffer.isBuffer(image).should.be.true;
+      var statsClipped = fs.statSync(tmp_dir+'/test-clipped.jpg');
+      image.length.should.be.eql(statsClipped.size);
```

Comment:
Maybe this one can also use `should.equal` for consistency? Or is there any specific reason of doing it this way?